db/migrate/20211229072937_create_meals.rb

```diff
@@ -1,6 +1,6 @@
-class CreateMenuItems < ActiveRecord::Migration[6.1]
+class CreateMeals < ActiveRecord::Migration[6.1]
   def change
-    create_table :menu_items do |t|
+    create_table :meals do |t|
```

Comment:
you already have this table on main branch. You have to create new one to rename this table